src/test/resources/TestUsers.csv

```diff
@@ -0,0 +1,4 @@
+"Username", "UUID"
+MHF_Alex,
+MHF_Steve,
+,
```

Comment:
Correct me if i'm wrong, but I don't think the CSV parser will complain about this?
```suggestion
,

```